src/region/RegionManager.tsx

```diff
@@ -36,7 +36,7 @@ export default class RegionManager implements BaseManager {
         rootLayerEl.setAttribute('data-resin-feature', 'annotations');
 
         // Insert the new layer element immediately after the reference element
-        return parentEl.insertBefore(rootLayerEl, referenceEl.nextSibling);
+        return parentEl.insertBefore(rootLayerEl, referenceEl);
```

Comment:
This is not safe, as it will likely impact region annotations on images.

Reply:
I'm trying to come up with an alternative solution to this .. would appreciate any pointers on where this logic could be moved.